code/game/objects/items/implants/implant_abductor.dm

```diff
@@ -5,22 +5,17 @@
 	icon_state = "implant"
 	activated = 1
 	var/obj/machinery/abductor/pad/home
-	var/cooldown = 30
+	var/cooldown = 60 SECONDS
+	var/on_cooldown
```

Comment:
I don't see this being set to true anywhere?

Reply:
This variable is set on line 18

Reply:
Swapped to the new system, so this is no longer relevant